client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts

```diff
@@ -337,7 +324,7 @@ const useContactQuestioning = (parameters: useContactQuestioningParameters): use
     };
 
     const checkForDuplicateIds = (idToCheck: string, interactedContactId: number) => {
-        return allContactedInteractions.findIndex((contact) => !idToCheck && contact.identificationNumber === idToCheck
```

Comment:
why did you delete it? I think it's Orel's fix for the null ids bug

Reply:
That makes the function to always return -1, the undefined check is also happening before calling the function